client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -110,13 +114,16 @@ const BrewRenderer = createClass({
 	},
 
 	renderPage : function(pageText, index){
-		return <div className='phb' id={`p${index + 1}`} dangerouslySetInnerHTML={{ __html: Markdown.render(pageText) }} key={index} />;
+		if(this.state.renderer == 'legacy')
+			return <div className='phb' id={`p${index + 1}`} dangerouslySetInnerHTML={{ __html: MarkdownLegacy.render(pageText) }} key={index} />;
+		else
+			return <div className='phb' id={`p${index + 1}`} dangerouslySetInnerHTML={{ __html: Markdown.render(pageText) }} key={index} />;
```

Comment:
```suggestion
                let HTML = "";
		if(this.state.renderer == 'legacy')
			HTML = MarkdownLegacy.render(pageText);
			
		else
			HTML = Markdown.render(pageText);
		return <div className='phb' id={`p${index + 1}`} dangerouslySetInnerHTML={{ __html: HTML }} key={index} />;
```

Reply:
Still tweaking this area a bit. This might change, but if not I will apply this.